plugin/documents.py

```diff
@@ -917,6 +923,27 @@ def _update_stored_selection_async(self) -> Tuple[Optional[sublime.Region], bool
         self._stored_selection = selection
         return changed_first_region, True
 
+    def _format_on_paste(self) -> None:
+        self.purge_changes_async()
+        clipboard_text = sublime.get_clipboard()
+        sel = self.view.sel()
+        split_clipboard_text = clipboard_text.split('\n')
+        multi_cursor_paste = len(split_clipboard_text) == len(sel) > 1
+        restore_regions = []
```

Comment:
naming: maybe call "original_selection"
Also I suppose it can be just `original_selection = list(sel)`